participants/scipy_018/doc/README.md

```diff
@@ -1,13 +1,15 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+The PR_tutorial package contains some simple functions to demonstrate the process of creating and merging a PR request.
+
+One function computes the factorial of a number:
 
     >>> from simple_functions import factorial
     >>> factorial(10)
     9
 
-and this other part does something.  I forget why that I did it:
+Another function computes the Fibonacci sequence:
 
     >>> fibonnaccci(100)
```

Comment:
Fibonacci (not "fibonnaccci") is misspelled here. Also, it would be helpful to know how to import this function.

Reply:
Good catch.